### DISCUSSION TITLE
[QUESTION]  does removing backup CRDs delete backups from the s3 storage?

### DISCUSSION THREAD

**Discussion by migs35323** [longhorn/longhorn#5317] [Q&A]
if the custom resource "Kind: backup" gets deleted in the k8s api (for example trough rancher or k8s dashboard), will the data get wiped from the backup store (minio s3)?

**Comment by joshimoo** [comment]
@migs35323 the backup custom resource handles the lifecycle of the backup files on the backupstore (S3/NFS) so if you delete the resource the backup will be deleted assuming the longhorn system is installed and active and a backupstore is configured.
